### DISCUSSION TITLE
Error 500 When trying to add image to profiles readme.md

### DISCUSSION THREAD

**Discussion by Caden-Cobb** [community/community#76602] [Repositories] [answered]
### Select Topic Area

Bug

### Body

So essentially I drag and drop the image I want on my profile's readme.md however after doing so it doesn't load in and prompts: ERROR 500. Any idea why? Thanks in advance.

**Comment by ddddami** [comment, marked as answer]
try a hard refresh and upload again?

**Reply by Caden-Cobb** [reply]
It seemed to fix itself overnight, ty though!
